Python/PRP/GUI/pyGUIButtonMod.cpp

```diff
@@ -0,0 +1,188 @@
+    "Remove all mouse-over animation keys from the button mod")
+{
+    self->fThis->clearMouseOverKeys();
+    Py_RETURN_NONE;
+}
+
+static PyMethodDef pyGUIButtonMod_Methods[] = {
+    pyGUIButtonMod_addAnim_method,
+    pyGUIButtonMod_delAnim_method,
+    pyGUIButtonMod_clearAnims_method,
+    pyGUIButtonMod_addMouseOver_method,
+    pyGUIButtonMod_delMouseOver_method,
+    pyGUIButtonMod_clearMouseOvers_method,
+    PY_METHOD_TERMINATOR
+};
+
+PY_GETSET_GETTER_DECL(GUIButtonMod, animationKeys) {
+    PyObject* list = PyTuple_New(self->fThis->getAnimationKeys().size());
+    for (size_t i = 0; i<self->fThis->getAnimationKeys().size(); i++)
+        PyTuple_SET_ITEM(list, i, pyPlasma_convert(self->fThis->getAnimationKeys()[i]));
+    return list;
+}
+PY_PROPERTY_SETTER_MSG(GUIButtonMod, animationKeys, "To add animations, use addAnim()")
+PY_PROPERTY_GETSET_DECL(GUIButtonMod, animationKeys)
+
+PY_GETSET_GETTER_DECL(GUIButtonMod, mouseOverKeys) {
+    PyObject* list = PyTuple_New(self->fThis->getMouseOverKeys().size());
+    for (size_t i = 0; i<self->fThis->getMouseOverKeys().size(); i++)
+        PyTuple_SET_ITEM(list, i, pyPlasma_convert(self->fThis->getMouseOverKeys()[i]));
+    return list;
+}
+PY_PROPERTY_SETTER_MSG(GUIButtonMod, mouseOverKeys, "To add mouse-over animations, use addMouseOver()")
+PY_PROPERTY_GETSET_DECL(GUIButtonMod, mouseOverKeys)
+
+PY_PROPERTY(ST::string, GUIButtonMod, animName, getAnimationName, setAnimationName)
+PY_PROPERTY(ST::string, GUIButtonMod, mouseOverAnimName, getMouseOverAnimName, setMouseOverAnimName)
+PY_PROPERTY(plKey, GUIButtonMod, draggable, getDraggable, setDraggable)
+PY_PROPERTY(int, GUIButtonMod, notifyType, getNotifyType, setNotifyType)
+
+static PyGetSetDef pyGUIButtonMod_GetSet[] = {
+    pyGUIButtonMod_animName_getset,
+    pyGUIButtonMod_mouseOverAnimName_getset,
+    pyGUIButtonMod_draggable_getset,
+    pyGUIButtonMod_notifyType_getset,
+    pyGUIButtonMod_animationKeys_getset,
+    pyGUIButtonMod_mouseOverKeys_getset,
+    PY_GETSET_TERMINATOR
+};
+
+PY_PLASMA_TYPE(GUIButtonMod, pfGUIButtonMod, "pfGUIButtonMod wrapper");
+
+PY_PLASMA_TYPE_INIT(GUIButtonMod) {
+    pyGUIButtonMod_Type.tp_new = pyGUIButtonMod_new;
+    pyGUIButtonMod_Type.tp_methods = pyGUIButtonMod_Methods;
+    pyGUIButtonMod_Type.tp_getset = pyGUIButtonMod_GetSet;
+    pyGUIButtonMod_Type.tp_base = &pyGUIControlMod_Type;
+    if (PyType_CheckAndReady(&pyGUIButtonMod_Type) < 0)
+        return NULL;
+
+    /* Konstants */
+    PY_TYPE_ADD_CONST(GUIButtonMod, "kNotifyOnUp", pfGUIButtonMod::kNotifyOnUp);
+    PY_TYPE_ADD_CONST(GUIButtonMod, "kNotifyOnDown", pfGUIButtonMod::kNotifyOnDown);
+    PY_TYPE_ADD_CONST(GUIButtonMod, "kNotifyOnUpAndDown", pfGUIButtonMod::kNotifyOnUpAndDown);
+
+    Py_INCREF(&pyGUIButtonMod_Type);
+    return (PyObject*)&pyGUIButtonMod_Type;
+}
+
+PY_PLASMA_IFC_METHODS(GUIButtonMod, pfGUIButtonMod)
+
+}
+
+extern "C" {
+
+PY_PLASMA_NEW(GUIMenuItem, pfGUIMenuItem)
```

Comment:
When putting multiple types in a single source file, please denote each section with a comment.  For example:
```c++
/* pyGUIButtonMod */
...

/* pyGUIMenuItem */
...
```